effort_controllers/src/joint_group_effort_controller.cpp

```diff
@@ -55,11 +55,14 @@ controller_interface::CallbackReturn JointGroupEffortController::on_deactivate(
   const rclcpp_lifecycle::State & previous_state)
 {
   auto ret = ForwardCommandController::on_deactivate(previous_state);
-
   // stop all joints
   for (auto & command_interface : command_interfaces_)
   {
-    command_interface.set_value(0.0);
+    if (!command_interface.set_value(0.0))
+    {
+      RCLCPP_WARN(get_node()->get_logger(), "Error while setting command interface value to 0.0");
+    }
+    return controller_interface::CallbackReturn::SUCCESS;
```

Comment:
this should go into the if branch, right?
```suggestion
    if (!command_interface.set_value(0.0))
    {
      RCLCPP_WARN(get_node()->get_logger(), "Error while setting command interface value to 0.0");
      return controller_interface::CallbackReturn::SUCCESS;
    }
```